backend/test/integration/pipeline_version_api_test.go

```diff
@@ -278,6 +278,37 @@ func (s *PipelineVersionApiTest) TestPipelineVersionAPI() {
 	var expectedWorkflow v1alpha1.Workflow
 	err = yaml.Unmarshal(expected, &expectedWorkflow)
 	assert.Equal(t, expectedWorkflow, *template)
+
+	/* ---------- Update pipeline default version ---------- */
+
+	pipelineVersionParams = uploadParams.NewUploadPipelineVersionParams()
+	pipelineID := "testId15242123"
+	pipelineVersionParams.SetPipelineid(&pipelineID)
+	argumentYAMLPipelineVersion, err = s.pipelineUploadClient.UploadPipelineVersion("../resources/arguments-parameters.yaml", pipelineVersionParams)
+	assert.Nil(t, err)
+	assert.Equal(t, "arguments-parameters.yaml", argumentYAMLPipelineVersion.Name)
+
+	argumentYAMLPipelineVersion, err = s.pipelineUploadClient.UploadPipelineVersion("../resources/arguments-parameters.yaml", pipelineVersionParams)
+	assert.Nil(t, err)
+	assert.Equal(t, "arguments-parameters.yaml", argumentYAMLPipelineVersion.Name)
+
+	versions, _, _, err := s.pipelineClient.ListPipelineVersions(&params.ListPipelineVersionsParams{ResourceKeyID: &pipelineID})
+	assert.Nil(t, err)
+	time.Sleep(1 * time.Second)
+	err = s.pipelineClient.UpdateDefaultVersion(&params.UpdatePipelineDefaultVersionParams{PipelineID: pipelineID,
+		VersionID: versions[0].ID})
+	assert.Nil(t, err)
+	filter := `
+		{
+			"op": 1,
+			"key": id,
+			"stringValue": "testId15242123"
+		}`
+	var pageSize int32 = 10
+	pipelineList, _, _, err := s.pipelineClient.List(&params.ListPipelinesParams{Filter: &filter, PageSize: &pageSize})
```

Comment:
If you want to get a pipeline by ID, the easiest way is to use `Get`:
https://github.com/kubeflow/pipelines/blob/f445c09f8181b520caf53340aec217ace13290ed/backend/src/common/client/api_server/pipeline_client.go#L71

Reply:
Great, will update.